internal/controllers/import_controller.go

```diff
@@ -273,38 +273,6 @@ func (r *CAPIImportReconciler) reconcileNormal(ctx context.Context, capiCluster
 	return ctrl.Result{}, nil
 }
 
-func (r *CAPIImportReconciler) shouldAutoImport(ctx context.Context, capiCluster *clusterv1.Cluster) (bool, error) {
-	log := log.FromContext(ctx)
-	log.V(2).Info("should we auto import the capi cluster", "name", capiCluster.Name, "namespace", capiCluster.Namespace)
-
-	// Check CAPI cluster for label first
-	hasLabel, autoImport := shouldImport(capiCluster)
-	if hasLabel && autoImport {
-		log.V(2).Info("Cluster contains import annotation")
```

Comment:
It is no longer an annotation, but a label

Reply:
It is a removed code, so should not matter?